include/benchmark/reporter.h

```diff
@@ -157,13 +157,18 @@ class BenchmarkReporter {
 // default reporter used by RunSpecifiedBenchmarks().
 class ConsoleReporter : public BenchmarkReporter {
  public:
+  explicit ConsoleReporter(bool color_output = true)
```

Comment:
one rather nice trick for avoiding boolean parameters (which can be confusing at the callsite) is to define an enum with two values.